protocol/wasmbinding/msg_plugin.go

```diff
@@ -48,6 +58,16 @@ func (m *CustomMessenger) DispatchMsg(ctx sdk.Context, contractAddr sdk.AccAddre
 		if contractMsg.DepositToSubaccount != nil {
 			return m.depositToSubaccount(ctx, contractAddr, contractMsg.DepositToSubaccount)
 		}
+		if contractMsg.WithdrawFromSubaccount != nil {
+			return m.withdrawFromSubaccount(ctx, contractAddr, contractMsg.WithdrawFromSubaccount)
```

Comment:
nit: overall, probably should think through where it makes sense to default an address to the `contractAddr` instead of taking in what's in the message, this extends to the design of the APIs as well. 
E.g.
- should a contract be allowed to withdraw from other addresess to other subaccounts?
- placing orders on other addresses?
- etc

Reply:
I think it's fine here to always pass in the `contractAddrs` here and have the `CustomMessenger` handle validations on `contractAddr`, and `WithdrawFromSubaccount.sender`, `WithdrawFromSubacocunt.receiver`.

See comment [below](https://github.com/dydxprotocol/v4-chain/pull/1612/files#r1633507953)